apps/sertra/components/ModCard.tsx

```diff
@@ -8,6 +9,14 @@ interface ModCard {
   icon_url: string;
 }
 
+interface modImageLoaderInputProps {
+  src: string;
+}
+
+const modImageLoader = ({ src }: modImageLoaderInputProps) => {
```

Comment:
Good points, still getting used to finding the types 👍 Seems like the Image thing doesn't work without a width, even though its not used in the loader. The example here is a bit weird for me also (https://nextjs.org/docs/api-reference/next/image#loader), since it's just slapping the resizing at the end of the url, which I guess should be a feature on the images hosting side 🤔

Anyways I do think, the latest version (soon to be updated to this branch), is fine enough for the Alpha.

Reply:
I guess the 3rd point in my previous comment was vaguely worded (too many "it" making it unclear what I'm referring to).

What I meant was that it looks the `loader` prop isn't needed. And since your implementation just returns the `src` parameter as is, I think defining the loader func and including the prop only adds noise. So I would remove the `loader` prop if the image works as expected without it.

Reply:
Ah yes, seems like the loader can be removed as long as it has that `unoptimized` as `true`, but I'm not sure why it works with it 🤷 I tried it without the loader, before adding the unoptimized, but back then it didn't work.

Might have something to do with the fact that "optimization" is probably configured  manually in the loader, depending on the hosting parties way of delivering "optimized" images? 🤷 Anyways should be good now

Reply:
I'm not thrilled with the amount of guesswork and shrugging emoji here. Created TS-1054 for figuring this out so it's not a blocker for this PR.